app/main/routes.py

```diff
@@ -77,8 +79,12 @@ def upload():
                 fund=Config.FUND_NAME,
             )
         else:
-            current_app.logger.info("Validation errors found during upload")
-
+            combined_error = {
+                key: [error[key] for error in validation_errors]
+                for key in ["cell_index", "description", "section", "sheet"]
+            }
+            if Config.ENABLE_VALIDATION_LOGGING:
+                current_app.logger.info(f"Validation errors found during upload: {combined_error}")
```

Comment:
We always want to log this "Validation errors found during upload", regardless of the flag. We just want the flag to toggle whether we include the `validation errors` as JSON objects